lib/pages/organizations.dart

```diff
@@ -1,44 +1,105 @@
+    "Georgia Domestic Violence Shelters",
+    "Greatist",
+    "Family Violence Prevention Fund",
+    "Men Stopping Violence",
+    "National Center on Domestic Violence, Trauma, & Mental Health",
+    "National Clearinghouse for the Defense of Battered Women",
+    "National Network to End Domestic Violence",
+  ];
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: HMAppBar(
         showLeading: true,
         showTrailing: true,
         leadingAction: () => Navigator.pop(context),
-        trailingAction: () => debugPrint("something goes here, check task in Zenhub"),
       ),
       body: Center(
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.center,
           children: <Widget>[
-            Text(
-                'Will be a list of organizations',
-                style: new TextStyle(
-                    color: Colors.deepPurpleAccent,
-                    fontSize: 20
-                )
+            Container(
+              padding: const EdgeInsets.fromLTRB(30.0, 30.0, 30.0, 0.0),
+              child: Text(
+                'Important Organizations',
+                style:
+                new TextStyle(color: purple.shade500, fontSize: 30, fontWeight: FontWeight.bold),
+              ),
             ),
-            ListView(
-              shrinkWrap: true,
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
               children: <Widget>[
-                ListTile(
-                  leading: Icon(Icons.label),
-                  title: Text('Organization 1'),
-                ),
-                ListTile(
-                  leading: Icon(Icons.label),
-                  title: Text('Organization 2'),
-                ),
-                ListTile(
-                  leading: Icon(Icons.label),
-                  title: Text('Organization 3'),
+                Align(
+                  alignment: Alignment.center,
+                  child: ListView.builder(
+                    padding: const EdgeInsets.fromLTRB(20.0, 10.0, 15.0, 10.0),
+                    shrinkWrap: true,
+                    itemCount: organizationUrl.length,
+                    itemBuilder: (BuildContext context, int index) {
+                      return Container(
+                        alignment: Alignment.center,
+                        padding: const EdgeInsets.all(20.0),
+                        child: GestureDetector(
+                          onTap: () => launch(organizationUrl[index]),
+                          child: Padding(
+                            padding: const EdgeInsets.only(left: 50.0),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.start,
+                              crossAxisAlignment: CrossAxisAlignment.center,
+                              children: <Widget>[
+                                Icon(Icons.laptop),
```

Comment:
I don't think this icon really adds anything, as it is used for each organization and serves more as a bullet point. I think it also takes up a lot of room. I would remove it.